NonSucking.Framework.Serialization.Advanced/BinarySpanWriter.cs

```diff
@@ -0,0 +1,265 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Text;
+using NonSucking.Framework.Serialization;
+
+namespace NonSucking.Framework.Serialization
+{
+    class ThrowHelper
+    {
+#if NETSTANDARD2_1_OR_GREATER
```

Comment:
This doesn't work for net 5, shouldn't it be
```suggestion
#if NETSTANDARD2_1_OR_GREATER || NETCOREAPP3_1_OR_GREATER
```